### DISCUSSION TITLE
pixi and $XDG_CONFIG_HOME on macOS

### DISCUSSION THREAD

**Discussion by anjos** [prefix-dev/pixi#1037] [Q&A] [answered]
My understanding is that `$XDG_CONFIG_HOME`, if unset, defaults to `$HOME/.config` [as per specifications](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html).

However, I see that on macOS pixi searches at `INFO pixi::config: Global config not found at /Users/userid/Library/Application Support/pixi/config.toml`

I checked the package [xdg-base-dirs](https://pypi.org/project/xdg-base-dirs/), and it reports `~/.config` as the default on macOS as well.

Would anyone know if this is this intended or a bug?

**Comment by ruben-arts** [comment]
Oh I would have expected the same. We're using this library: https://docs.rs/dirs/5.0.1/dirs/fn.config_dir.html

Where they specify it in the location you get. I'm not sure why there are differences between mac and linux.

**Reply by anjos** [reply]
Well, that library is not clailming to be XDG compatible.  However, [pixi's documentation is](https://pixi.sh/latest/advanced/global_configuration/).  Personally, I'd be in favour of XDG compatibility.  In practice you'd have to choose between fixing the documentation or changing the library.

Should I open an issue?

**Reply by chawyehsu** [reply]
It's ok to open an issue I think, and it's also simple to support this by adding `$HOME/.config` check to the [config loader](https://github.com/prefix-dev/pixi/blob/8de97fb606891e9abe16c7d49425721a56f7ad89/src/config.rs#L163). The strategy the `dirs` library provides is actually using platform-recommended implementation. I could help to submit a patch to address this when you open the issue.

**Reply by anjos** [reply, marked as answer]
Thanks for the support: the issue is now created (https://github.com/prefix-dev/pixi/issues/1049)

**Comment by pavelzw** [comment]
Related to #276

**Reply by ruben-arts** [reply]
Hmm in this case it doesn't really make sense to configure the global configuration dir in the global configuration :thinking:
